src/drivers/fluid_coremidi.c

```diff
@@ -67,6 +69,31 @@ void fluid_coremidi_driver_settings(fluid_settings_t *settings)
     fluid_settings_register_str(settings, "midi.coremidi.id", "pid", 0);
 }
 
+static void fluid_coremidi_autoconnect(fluid_coremidi_driver_t *dev, MIDIPortRef input_port)
+{
+    unsigned int i;
+    int source_count = MIDIGetNumberOfSources();
+    for(i = 0; i < source_count; ++i)
```

Comment:
Now, the  comparision `i < source_count` differs in signedness. Prefer using int.